config.md

```diff
@@ -211,7 +211,14 @@ For Linux-based systems, the `process` object supports the following process-spe
     This is a per-process setting, where as [`disableOOMKiller`](config-linux.md#memory) is scoped for a memory cgroup.
     For more information on how these two settings work together, see [the memory cgroup documentation section 10. OOM Contol][cgroup-v1-memory_2].
 * **`selinuxLabel`** (string, OPTIONAL) specifies the SELinux label for the process.
-    For more information about SELinux, see  [SELinux documentation][selinux].
+    For more information about SELinux, see [SELinux documentation][selinux].
+* **`landlock`** (object, OPTIONAL) specifies the Landlock unprivileged access control settings for the container process.
+    For more information about Landlock, see [Landlock documentation][landlock].
+    `landlock` contains the following properties:
+
+    * **`ruleset`** (object, OPTIONAL) the `ruleset` field identifies a set of rules (i.e., actions on objects) that need to be handled (i.e., restricted).
+    * **`rules`** (array of objects, OPTIONAL) the `rules` field specifies the security policies (i.e., actions allowed on objects) to be added to an existing ruleset
+    * **`abi`** (object, OPTIONAL) the `abi` field defines the specific Landlock ABI version.
```

Comment:
About the "abi" field, it is not explained how this will be used. Is it a requirement to only enforce the Landlock rules with a kernel supporting this version? Is it a requirement for a minimal version?

The ABI version should be used by the container runtime to know if a set of rules is supported by the kernel, which should then help to only enforce those that are indeed supported and ignores unsupported features (i.e. best-effort security).

I'm not sure it makes sense to expose this in the configuration.

Reply:
I personally agree w/ the best-effort security approach and I'm currently implementing this way (i.e., if not supported, just drop a warning and continue interpreting the configuration).
Added with more elaborations on the usage of this `abi` field. IMO, it's fine to expose this in the config. Feedback is welcome.

Reply:
Are the `abi` and `handledAccessFS` settings trying to solve the same problem here? In the `golandlock` module, picking an ABI version is just a shortcut for asking for the maximum set of `handledAccessFS` possible under that ABI.

Reply:
@gnoack Thanks! Great question.

Yes, from the current `golandlock` implementation, the `abi` maps to the max set of `handledAccessFS` possible under it. I.e., they may try to solve the same problem at this moment.

For the new `abi`s in the future, should they normally point to sets of features to be supported by the kernel and those might go beyond `handledAccessFS`? I'm not sure. I'll leave the question here to @l0kod.